plugins/sync.js

```diff
@@ -533,22 +593,27 @@ window.plugin.sync.Authorizer = function(options) {
   this.isAuthorizing = this.isAuthorizing.bind(this);
   this.authorize = this.authorize.bind(this);
 };
-
+// values that are static at the moment
 window.plugin.sync.Authorizer.prototype.CLIENT_ID = '1099227387115-osrmhfh1i6dto7v7npk4dcpog1cnljtb.apps.googleusercontent.com';
 window.plugin.sync.Authorizer.prototype.SCOPES = 'https://www.googleapis.com/auth/drive.file';
 
+// method to tell if SYNC is authorized
```

Comment:
I would prefer javadoc style 
```suggestion
/**
 * aks if SYNC is authorized
 * @return true if authorized
 **/
```